samples/vision/object-detection/create-dataset.v1beta1.js

```diff
@@ -0,0 +1,74 @@
+/**
+ * Copyright 2019, Google LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+`use strict`;
+function main(
+  projectId = 'YOUR_PROJECT_ID',
+  computeRegion = 'YOUR_REGION_NAME',
+  datasetName = 'YOUR_DATASET_NAME'
+) {
+  // [START automl_vision_object_detection_create_dataset]
+  /**
+   * Demonstrates using the AutoML client to create a dataset.
+   * TODO(developer): Uncomment the following lines before running the sample.
+   */
+  // const projectId = '[PROJECT_ID]' e.g., "my-gcloud-project";
+  // const computeRegion = '[REGION_NAME]' e.g., "us-central1";
+  // const datasetName = '[DATASET_NAME]' e.g., "myDataset";
+
+  //Imports the Google Cloud Automl library
+  const {AutomlClient} = require('@google-cloud/automl').v1beta1;
+
+  // Instantiates a client
+  const automlClient = new AutomlClient();
+  const util = require(`util`);
+
+  async function createDataset() {
+    // A resource that represents Google Cloud Platform location.
+    const projectLocation = automlClient.locationPath(projectId, computeRegion);
+
+    // Set dataset name and metadata.
+    const myDataset = {
+      displayName: datasetName,
+      imageObjectDetectionDatasetMetadata: {},
+    };
+
+    // Create a dataset with the dataset metadata in the region.
+    automlClient
```

Comment:
Please use async/await here, and all throughout

Reply:
fixed in the latest commits